### DISCUSSION TITLE
Postgres array column / Tags field

### DISCUSSION THREAD

**Discussion by acetinick** [avo-hq/avo#360] [Feature requests]
Hey, does this library support array columns for editing. Like string arrays as a tag input?

**Comment by adrianthedev** [comment]
Not yet, but that's a good field to have. 
I'm thinking of something like [tagify](https://github.com/yairEO/tagify)
Have you seen a nice implementation of a tags input anywhere?

**Reply by acetinick** [reply]
I been rolling out https://tom-select.js.org/ across my app,  big fan of it, and you can even include it as a esm module so dont have to worry about npm.

Would be cool to see array columns supported.   Going to start trialing the product to replace active admin soon

**Comment by adrianthedev** [comment]
https://github.com/avo-hq/avo/pull/847 adds the tags field.
